CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ x.x.x Release notes (yyyy-MM-dd)
   types.
 * Remove `RLMPropertyTypeArray` in favor of a separate bool `array` property on
   `RLMProperty`/`Property`.
+* Realm Swift APIs that filter or look up the index of an object based on a
```

Comment:
Is this really a breaking change rather than a bug fix? From #4197 it sounds like passing `none` through doesn't do anything useful at present, while it will after this change.